LambdaEngine/Source/Game/ECS/Systems/Networking/ClientSystem.cpp

```diff
@@ -117,18 +117,19 @@ namespace LambdaEngine
 			m_pClient->SendReliable(pPacket);
 
 			ECSCore* pECS = ECSCore::GetInstance();
-			const auto* pPositionComponents = pECS->GetComponentArray<PositionComponent>();
+			auto* pControllableComponent = pECS->GetComponentArray<ControllableComponent>();
 
-			if (!pPositionComponents)
+			if (!pControllableComponent)
 				return;
 
-			const PositionComponent& positionComponent = pPositionComponents->GetData(GetEntityPlayer());
+			ControllableComponent& controllableComponent = pControllableComponent->GetData(GetEntityPlayer());
```

Comment:
Should be const since we don't change any values of the Component
```suggestion
			const ControllableComponent& controllableComponent = pControllableComponent->GetData(GetEntityPlayer());
```